roles/installer/templates/statefulsets/postgres.yaml.j2

```diff
@@ -137,7 +137,15 @@ spec:
 {% if postgres_storage_class is defined %}
         storageClassName: '{{ postgres_storage_class }}'
 {% endif %}
-        resources: {{ postgres_storage_requirements }}
+{% if postgres_storage_requirements.limit or postgres_storage_requirements.requests %}
+        resources:
+{% if postgres_storage_requirements.limit %}
+          limit: {{ postgres_storage_requirements.limit }}
```

Comment:
Should we even allow the customer to set this? When I looked up if this field had any affect on pvc's, everything came up no

Reply:
it should be fine...